### PR TITLE
Fix hostGroup conversion logic for v1beta2

### PR DESCRIPTION
**Pull request by luhi-DT**
## Description
Ticket: https://dt-rnd.atlassian.net/browse/K8S-10198

Currently the hostGroup parameter is not converted from v1beta1 to v1beta2.

## How can this be tested?
Install Operator -> Apply a v1beta1 Dynakube with spec.oneAgent.hostGroup -> check if the converted v1beta2 Dynakube also has hostGroup set

### CODE REVIEW

File: pkg/api/v1beta1/dynakube/convert_from.go [hunk 1]
```diff
@@ -38,6 +38,7 @@ func (dst *DynaKube) fromBase(src *v1beta2.DynaKube) {
 }
 
 func (dst *DynaKube) fromOneAgentSpec(src *v1beta2.DynaKube) {
+	dst.Spec.OneAgent.HostGroup = src.Spec.OneAgent.HostGroup
```
Comment (andriisoldatenko) [inline pkg/api/v1beta1/dynakube/convert_from.go:41]: add unit-test 🥺 please

Comment (0sewa0) [reply pkg/api/v1beta1/dynakube/convert_from.go:41]: ☝️

Comment (luhi-DT) [reply pkg/api/v1beta1/dynakube/convert_from.go:41]: forgot to git add...... the file

**Review by andriisoldatenko — APPROVED** [review]

**Review by wepudt — APPROVED** [review]
LGTM

deployed v1.beta1 DK with hostGroup set -> v1beta2 DK has same hostGroup set.
also tested v1beta2 DK converted to v1beta1. conversion works in both directions.

**Pull request merged**